tekcamp.sql

```diff
@@ -0,0 +1,256 @@
+-- Step 1
+show databases;
+select * from students;
+
+-- Step 2
+show databases;
+
+create database tc1_tyler_clements_;
+
+-- Step 3/4
+show databases;
+
+use tc1_tyler_clements_;
+show tables;
+
+create table tekcampers as
+select * from students.tekcamp01;
+
+select * from tekcampers;
+
+-- Step 5
+alter table tekcampers
+add primary key (id);
+
+insert into tekcampers (id, first_name, last_name, gender, education)
+values 
+(28, 'Aaron', 'White', 'M', NULL), 
+(29, 'Cody', 'Clark', 'M', 'Some College'), 
+(30, 'Devon', 'Brewster', 'M', 'Some College'), 
+(31, 'Gotham', 'Katta', 'M', 'Bachelor’s Degree'), 
+(32, 'John', 'Bozarov', 'M', 'Bachelors degree'), 
+(33, 'John', 'Kol', 'M', NULL), 
+(34, 'Justin', 'Cheng', 'M', 'Some College'), 
+(35, 'Kevin', 'Keesee', 'M', 'High School Diploma'), 
+(36, 'Monica', 'Howard', 'F', NULL), 
+(37, 'Tyler', 'Clements', 'M', 'Bachelors Degree'),
+(38, 'Vimala', 'Murthy', 'F', 'Masters Degree');
+
+select * from tekcampers;
+
+-- Step 6
+delete from tekcampers 
+where 
+id in (9, 20, 26);
+
+select * from tekcampers;
+
+-- Step 7
+update tekcampers
```

Comment:
having lookup tables for shared/repeated values would help with consistency (and is more relational)